nominees/openrims.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "OpenRIMS",
+  "aliases": [
+    "Pharmadex"
+  ],
+  "description": "Digital Suite for Regulatory Authorities",
+  "website": "https://openrims.org",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://www.gnu.org/licenses/gpl-3.0.en.html"
```

Comment:
Invalid license URL. This should be a link to where the license is indicated for this project's repository. Thanks.